docs/getting_started/tutorial.md

```diff
@@ -118,6 +118,10 @@ If everything worked, <http://127.0.0.1:8000> will show you a welcome page:
 
 ![Browser screenshot of "Welcome to your new Wagtail site!" page, with teal egg above the title, and links to different resources. The page is shown inside a browswer tab, with browser URL bar at the top](../_static/images/tutorial/tutorial_1.png)
 
+```{note}
+Throughout this tutorial we will use `http://127.0.0.1:8000` but depending on your setup, this could be `http://localhost:8000/` or a different IP address or port. Please read the console output of `manage.py runserver` if you get stuck here. It will tell you what the IP / URL is of your site running locally.
```

Comment:
> Please read the console output of `manage.py runserver` if you get stuck here.  It will tell you what the IP / URL is of your site running locally.

How about: 

Please read the console output of `manage.py runserver` to determine the correct url for your local site.

Reply:
@cnk This seems to be a more direct and self-explanatory suggestion. In this case, I would make the change right away.